src/View/Article/list.html.twig

```diff
@@ -0,0 +1,47 @@
+{% extends 'layout.html.twig' %}
+{%block title %}{{ parent() }} - Article{% endblock %}
+{% block banner %} <div class="banner2"></div>{% endblock %}
+{% block content %}
+<section class="product">
+    <!-- Product -->
+    <div class="bg0 m-t-23 p-b-140">
+        <div class="container">
+            <!--Search bar-->
+            <div class="row d-flex justify-content-center bd-highlight mb-3  search_product_bar">
+                <nav class="m-b-100 search_bar col-md-8 col-lg-5  navbar navbar-light">
+                    <form class="search_bar form-inline" method="get"  action="">
+                        <input class="search_bar form-control mr-sm-2" type="search" name="search" placeholder="🔍 " aria-label="Search">
+                        <input type="hidden" name="category" value="DVD">
+                    </form>
+                </nav>
+            </div>
+        </div>
+    </div>
+    <!-- Block2 -->
+    <div class="container">
+        <div class="row ">
+            <div class="col-lg-12 bold cl13 fs-30 d-flex justify-content-center">{{ error.category }} {{ error.toomuch }}</div>
+            {% for article in article %}
```

Comment:
et là ça ne peut pas fonctionner, c'est {% for article in articles %} normalement